src/main/java/com/mineboundteam/minebound/event/Events.java

```diff
@@ -0,0 +1,54 @@
+package com.mineboundteam.minebound.event;
+
+import com.mineboundteam.minebound.MineBound;
+import net.minecraft.world.damagesource.DamageSource;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.EquipmentSlot;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.Items;
+import net.minecraftforge.event.entity.living.LivingAttackEvent;
+import net.minecraftforge.event.entity.living.LivingHurtEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod.EventBusSubscriber(modid = MineBound.MOD_ID)
+public class Events {
```

Comment:
For organization purposes, please rename this to DamageEvent or something similar

Reply:
Done. Renamed to PlayerDamageEvent.